pkg/vault/client_test.go

```diff
@@ -118,63 +115,6 @@ func TestGetKV1Secret(t *testing.T) {
 	})
 }
 
-func TestWriteKvSecret(t *testing.T) {
```

Comment:
why to remove it?

Reply:
it's not unit but integration test. And the same is tested here: https://github.com/SAP/jenkins-library/blob/d0e205d018238d00d4e69a9ff4e0b3e3d31b3280/integration/integration_vault_test.go#L71